java/ql/src/Security/CWE/CWE-730/PolynomialReDoS.qhelp

```diff
@@ -103,6 +101,33 @@
 
 	</example>
 
+    <example>
+        <p>
+            Sometimes it is unclear how a regular expression can be rewritten to
+            avoid the problem. In such cases, it often suffices to limit the
+            length of the input string. For instance, the following complicated 
```

Comment:
I'd maybe say avoid saying 'complicated', I think it has negative connotations that don't really fit the docs tone. You could say 'complex' if you think it's needed?

```suggestion
            length of the input string. For instance, the following 
```